models/articles_models.js

```diff
@@ -0,0 +1,17 @@
+const db = require('../db/connection');
+
+exports.fetchArticleByID = (id) => {
+    if(isNaN(id)){
+        return Promise.reject({status:400, msg:"Invalid article id"})
+    }
+    const idQuery = `WHERE article_id = ${id}`
+    return db
+    .query(`SELECT * FROM articles ${idQuery}`)
```

Comment:
To protect against SQL injection, we'd want to make use of a [parameterized query](https://notes.northcoders.com/courses/js-back-end/node-postgres) for article_id.